src/main/java/com/mongodb/quickstart/models/Grade.java

```diff
@@ -0,0 +1,78 @@
+package com.mongodb.quickstart.models;
+
+import org.bson.types.ObjectId;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Grade {
+
+    private ObjectId id;
+    private Double student_id;
+    private Double class_id;
+    private List<Score> scores;
+
+    public ObjectId getId() {
+        return id;
+    }
+
+    public Grade setId(ObjectId id) {
+        this.id = id;
+        return this;
+    }
+
+    public Double getStudent_id() {
```

Comment:
From a Java perspective this is a non-standard naming convention.  Consider `getStudentId` and use `@BsonProperty` if you want the field name to be different in the database itself.

Reply:
Updated + I will add a comment in the blog post about it.